### PR TITLE
KP description update

### PR DESCRIPTION
**Pull request by Dajamante**

### CODE REVIEW

File: ferrocene/doc/internal-procedures/src/known-problems.rst [hunk 1]
```diff
@@ -71,17 +79,10 @@ heading `Workaround`
 The `Workaround` section describes how to avoid running into the issue.
 This usually consists of rules that forbid certain code patterns.
 
-heading `Detection`
```
Comment (Veykril) [inline ferrocene/doc/internal-procedures/src/known-problems.rst:74]: Why are we removing this? Our KPs still have this section

Comment (Dajamante) [reply ferrocene/doc/internal-procedures/src/known-problems.rst:74]: Sorry! I don't know why I thought those two sections had been merged together....

File: ferrocene/doc/internal-procedures/src/known-problems.rst [hunk 2]
```diff
@@ -59,6 +59,14 @@ The `fixed-commit` frontmatter lists the hash of the commit(s) that fixed the
 issue if it has already been fixed.
 Leave it blank if the issue is still open.
 
+frontmatter `duplicate`
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The duplicate field is optional and marks whether a new problem is a duplicate 
+of an older one.
+`introduced-commit` and/or `fixed-commit` may be different; the problem is marked as duplicate if 
+it is marked as duplicate upstream.
```
Comment (Veykril) [inline ferrocene/doc/internal-procedures/src/known-problems.rst:68]:
```suggestion
it is considered as duplicate upstream.
```
wording nit, the Rust project has no actual way of "marking" duplicates. They just say duplicate of x and close it (or maybe even leave it open)

**Review by Veykril — APPROVED** [review]

**Comment by Veykril** [comment]
bors merge

**Comment by tshepang** [comment]
can you remove the merge commit

**Comment by Dajamante** [comment]
will reopen as I can't get rid of the merge commit

**Pull request closed without merging**